app/src/main/java/com/example/sopt_assignment_dabin/HomeActivity.kt

```diff
@@ -0,0 +1,43 @@
+package com.example.sopt_assignment_dabin
+
+import android.app.Activity
+import android.content.Intent
+import android.net.Uri
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.text.Editable
+import android.view.animation.AnimationUtils
+import com.bumptech.glide.Glide
+import com.example.sopt_assignment_dabin.databinding.ActivityHomeMainBinding
+
+
+class HomeActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val binding = ActivityHomeMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        //애니메이션
+        fun setAnimation() {
+            val animation = AnimationUtils.loadAnimation(this, R.anim.fade_in)
+            val animation2 = AnimationUtils.loadAnimation(this, R.anim.fade_out)
+            binding.imageView.startAnimation(animation2)
+            binding.tvTitleHome.startAnimation(animation2)
+            binding.imageView.startAnimation(animation)
+            binding.tvTitleHome.startAnimation(animation)
```

Comment:
스코프 함수를 활용해서
```suggestion
            binding.apply {
            imageView.startAnimation(animation2)
            tvTitleHome.startAnimation(animation2)
            imageView.startAnimation(animation)
            tvTitleHome.startAnimation(animation)
            }
```
이런식으로 간결하게 표현할 수도 있을 것 같아요~!!